oc-chef-pedant/lib/pedant/rspec/search_util.rb

```diff
@@ -708,9 +708,11 @@ def direct_solr_query(type, query)
         # Chef Object type (in addition to filtering based on org).  We
         # don't have an easy way to access the org's guid in the tests, so
         # I'm not using that. In any event, the following query works.
+        auth = Base64.strict_encode64("admin:admin")
         url = "#{Pedant::Config.search_server}#{Pedant::Config.search_url_fmt}" % {:type => CGI.escape(type), :query => CGI.escape(query)}
         headers = {
-          "Accept" => "application/json"
+          "Accept" => "application/json",
+          "Authorization" => "Basic #{auth}"
```

Comment:
The additional headers will not work for elasticsearch, correct? They should be included only if the provider is elasticsearch.

Reply:
we probably need something like maybe_add_headers(), perhaps in the helper that will add the Auth header for opensearch search provider only.